cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -772,6 +774,32 @@ func (k *KubernetesClient) mapPersistentVolumeClaims(group, version, kind, name,
 	}, nil
 }
 
+func (k *KubernetesClient) mapPersistentVolumes(group, version, kind, name, namespace string) (*dto.PersistentVolume, error) {
+	persistentvolume, err := k.clientset.CoreV1().PersistentVolumes().Get(context.Background(), name, metav1.GetOptions{})
```

Comment:
Can you rename this one to `persistentVolume`?

Reply:
fixed as suggested